src/components/UserEventLocation/UserEventLocation.tsx

```diff
@@ -1,39 +1,59 @@
+import { useEffect } from 'react';
 import { ReactSVG } from 'react-svg';
 import styles from './UserEventLocation.module.scss';
 import { LectureEvent } from '../../domain/lectureEvent';
 import busIcon from '../../assets/icons/busIcon.svg';
 import carIcon from '../../assets/icons/carIcon.svg';
 import { Faculty, generateGoogleMapsLink } from '../../helpers/googleMaps';
```

Comment:
change the file name to also reflect that this is not just for google maps but for navigation in general so a good name would be "navigation.ts"